### PR TITLE
Update open-in-json-hero extension

### PR DESCRIPTION
**Pull request by gjthompson1**
## Description

Want to add the ability to set the tab when loading the web app.

## Screencast
 
Screenshot 2024-07-04 at 13 17 07

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Review by mil3na — APPROVED** [review]
Hi!
Thank you for your contribution! My name is Milena and I will be reviewing it today.

LGTM!

File: extensions/open-in-json-hero/package.json [hunk 1]
```diff
@@ -23,6 +24,14 @@
     }
   ],
   "preferences": [
+    {
+      "name": "tab",
+      "description": "Currently you can leave this blank or choose /editor or /tree",
+      "type": "textfield",
+      "required": false,
+      "default": "/editor",
```
Comment (mil3na) [inline extensions/open-in-json-hero/package.json:32]: Maybe, if the options are limited and pre-defined, it should be a dropdown option, not a textfield, right?

**Review by mil3na — CHANGES_REQUESTED** [review]
Hi!
Thank you for your contribution! My name is Milena and I will be reviewing it today.

File: extensions/open-in-json-hero/package.json (hunk 1 above)
Comment (gjthompson1) [reply extensions/open-in-json-hero/package.json:32]: Yeh I had that originally but changed it to this as to be more flexible. If the site changes.

Comment (mil3na) [reply extensions/open-in-json-hero/package.json:32]: there's is also no validation to see if the path is valid. Either you add the dropdown or add validation :)

**Comment by mil3na** [comment]
Hello! Do you still want to go forward with this change?

**Pull request closed without merging**